setup.py

```diff
@@ -57,7 +61,6 @@ def read_version():
         'Issue Tracker': 'https://github.com/MasterSergius/acachecontrol/issues',
     },
     python_requires='>=3.6',
-    install_requires=[
-        'aiohttp'
-    ]
+    tests_require=test_requires,
+    install_requires=requires,
```

Comment:
IMHO, this should follow this strategy https://packaging.python.org/discussions/install-requires-vs-requirements/

Reply:
Do we expect to have different dependencies in `setup.py` and `requirements.txt`?

Reply:
Whereas install_requires metadata is automatically analyzed by pip during an install, requirements files are not, and only are used when a user specifically installs them using pip install -r.